ludwig/utils/torch_utils.py

```diff
@@ -271,6 +271,7 @@ def __init__(
         bias_initializer="zeros",
     ):
         super().__init__()
+        self.input_size_debug, self.output_size_debug = input_size, output_size
```

Comment:
Do we mean to save these as `input_size_debug` and `output_size_debug`?

Reply:
my bad, these are for debugging too